src/pages/index.tsx

```diff
@@ -412,7 +416,7 @@ export default function Home() {
 
                 <div className="flex flex-col w-[413.33px] h-[508px] items-center relative">
                   <div className="top-10 relative">
-                    <Image src={cardImage} height={300} width={300} alt="alt" />
+                    <Image src={`${cardImage || content.image[0]}`}  height={300} width={300} alt="alt" />
```

Comment:
This seems a bit confusing to me, why is this wrapped in a template literal?

Reply:
I thought src could only take in the form of a string.